cos_registration_agent/cli.py

```diff
@@ -247,10 +257,21 @@ def main():
                     loki_alert_rule_files=args.device_loki_alert_rule_files,
                     prometheus_alert_rule_files=args.device_prometheus_alert_rule_files,
                 )
+                logger.debug("YOOOO")
```

Comment:
wazzup :stuck_out_tongue_closed_eyes:

Reply:
removed in [ab714d1](https://github.com/canonical/cos-registration-agent/pull/53/commits/ab714d14c07e5f357697e4f86d647f675d2fa9b2)